src/bot/scrapper/scrapper.ts

```diff
@@ -1,26 +1,27 @@
 import puppeteer from "puppeteer";
 import {Page, Browser} from "puppeteer";
 
-interface ScrapperOptions {
+export interface ScrapperOptions {
     searchValue: string;
     maxRecords: number;
 }
 
 export class Scrapper implements ScrapperOptions {
     searchValue: string;
     maxRecords: number;
+    private browser: Browser
 
     constructor(options: ScrapperOptions) {
         this.searchValue = options.searchValue;
         this.maxRecords = options.maxRecords;
     }
 
     async initializePage() {
-        const browser = await puppeteer.launch();
-        return browser.newPage();
+        this.browser = await puppeteer.launch({headless: false});
```

Comment:
**question:** I guess `headless: false` is more for debugging purposes. Or do you need it for some other specific reason?

Reply:
yes, because 'indeed' has safeguards against more automated requests and you have to confirm that you are not a robot

Reply:
Have you tried this? https://www.npmjs.com/package/puppeteer-extra-plugin-stealth

Reply:
no, but I'm going to try it now :) thanks